### PR TITLE
Pass showIndex to NestedFormInput

### PR DESCRIPTION
**Pull request by niuyunsi**

### CODE REVIEW

File: src/components/NestedForm/NestedFormInput.tsx [hunk 1]
```diff
@@ -53,6 +56,8 @@ export class NestedFormInput extends React.PureComponent<data.IGenericFieldInput
 
     private renderEntry(entry: any, index: number) {
         const showDeleteButton = this.props.value.length !== 1;
+        const showIndex = this.props.showIndex !== undefined ? this.props.showIndex : true;
```
Comment (linweigao) [inline src/components/NestedForm/NestedFormInput.tsx]: undefined means false, right? why do you need this check?

Comment (niuyunsi) [reply src/components/NestedForm/NestedFormInput.tsx]: my thought was to check if showIndex is defined and passed to NestedFormInput, if showIndex is not defined then show index at default.

Comment (linweigao) [reply src/components/NestedForm/NestedFormInput.tsx]: Gotcha. Then I would suggest to use defaultProps.

Comment (niuyunsi) [reply src/components/NestedForm/NestedFormInput.tsx]: ok, I will change to use defaultProps

**Comment by linweigao** [comment]
You cannot check-in gigaform change first without this. Gigaform-fe cannot build now.

I have discussed with PM with showIndex option. The decision is not to let users decide. It should show index always.

**Comment by niuyunsi** [comment]
ok, I just sent a CL to remove show index option.

**Pull request closed without merging**